ethereum/eth/src/test/java/tech/pegasys/pantheon/ethereum/eth/sync/BlockPropagationManagerTest.java

```diff
@@ -458,11 +461,13 @@ public void ignoresOldNewBlockAnnouncement() {
     assertThat(blockchain.contains(oldBlock.getHash())).isFalse();
   }
 
+  @SuppressWarnings("unchecked")
   @Test
   public void purgesOldBlocks() {
     final int oldBlocksToImport = 3;
     syncConfig =
         SynchronizerConfiguration.builder().blockPropagationRange(-oldBlocksToImport, 5).build();
+    BlockBroadcaster<Void> blockBroadcaster = mock(BlockBroadcaster.class);
```

Comment:
We should be able to make the `blockBroadcaster` from `setUp` a field and just re-use it here.  Which then avoids the need for the `@SuppressWarnings("unchecked")` on this method.